index.html

```diff
@@ -0,0 +1,161 @@
+        <div class="catalog">
+            <p class="block-title tac mx-auto">Our friends who are looking for a house</p>
+            <div class="pets">
+                <div class="petcard">
+                    <img src="./img/pets-katrine.png" alt="katrine">
+                    <p>Katrine</p>
+                    <a href="">Learn more</a>
+                </div>
+                <div class="petcard">
+                    <img src="./img/pets-jennifer.png" alt="jennifer">
+                    <p>Jennifer</p>
+                    <a href="">Learn more</a>
+                </div>
+                <div class="petcard">
+                    <img src="./img/pets-woody.png" alt="woody">
+                    <p>Woody</p>
+                    <a href="">Learn more</a>
+                </div>
+            </div>
+            <button class="button d-block mx-auto mt-60 mb-100">Get to know the rest</button>
+        </div>
+        <div class="help">
+            <p class="block-title tac mx-auto">
+                How you can help our shelter
+            </p>
+            <div class="itemsfrow">
+                <div class="card">
+                    <img src="./img/icon-pet-food.png" alt="">
+                    <p>Pet food</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-transportation.png" alt="">
+                    <p>Transportation</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-toys.png" alt="">
+                    <p>Toys</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-bowls-and-cups.png" alt="">
+                    <p>Bowls and cups</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-shampoos.png" alt="">
+                    <p>Shampoos</p>
+                </div>
+            </div>
+            <div class="itemssrow">
+                <div class="card">
+                    <img src="./img/icon-vitamins.png" alt="">
+                    <p>Vitamins</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-medicines.png" alt="">
+                    <p>Medicines</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-collars-leashes.png" alt="">
+                    <p>Collars/leashes</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-sleeping-area.png" alt="">
+                    <p>Sleeping areas</p>
+                </div>
+            </div>
+        </div>
+        <div class="donation">
+            <div>
+                <img src="./img/donation-dog.png" alt="">
+            </div>
+            <div class="inform">
+                <p>You can also <br> make a donation</p>
+                <p class="bank">Name of the bank / Type of bank account</p>
```

Comment:
тут надо посмотреть, не можем ли мы использовать какой-то типовой класс
Само название класса не очень подходящее
```suggestion
                <p class="??">Name of the bank / Type of bank account</p>
```